index.js

```diff
@@ -60,6 +72,13 @@ function isObject(obj) {
   return typeOf(obj) === 'object' || typeOf(obj) === 'function';
 }
 
+/**
+ * Returns true if the object is an array.
+ */
+function isArray(obj) {
```

Comment:
Wouldn't it be easier just to use `Array.isArray`?

Reply:
The author *should* be using the `is-array` package.